src/python/grpcio/grpc/experimental/aio/_interceptor.py

```diff
@@ -248,7 +248,7 @@ async def intercept_stream_stream(
 
 
 class InterceptedCall:
-    """Base implementation for all intecepted call arities.
+    """Base implementation for all intercepted call arities.
```

Comment:
Thanks for fixing the typo. In any case seems that within the `_interceptror.py` file there are some reference to the old Metadata type like this [1] which would need to be changed, the same for the `ClientCallDetaills` which is still using an `Optional[MetadataType]` should this be changed to `Metadata`? [2]


[1] https://github.com/grpc/grpc/pull/23045/files#diff-173e8bd5d2219ba109f86237680068a4R383
[2] https://github.com/grpc/grpc/blob/master/src/python/grpcio/grpc/experimental/aio/_interceptor.py#L85

Reply:
One last thing, if the input from the user hasn't  changed, so its still providing the metadata for te call using a tuple I wold not change either the `ClientCallDetails` since this is an interface used by the interceptors for mutating the metadata provided by the user.